omnigibson/simulator.py

```diff
@@ -461,6 +515,57 @@ def viewer_width(self, width):
             """
             self._viewer_camera.image_width = width
 
+        def add_ground_plane(self, floor_plane_visible=True, floor_plane_color=None):
```

Comment:
normally this was done per-scene. I assume that we're going to enforce some limitations of the parallelization of envs as a result of this? e.g: can't have an empty scene and InteractiveScene simultaneously

Reply:
We were actually talking about this on Monday. We could potentially have one ground plane for each scene, but we will need to avoid collision checks between these planes as they would all be colliding with each other. Last time I checked with Cem, we are probably okay with multiple-envs only supporting the same scene for now. But we can definitely chat more about this @cgokmen

Reply:
Are we doing the collision group thing in a follow up?

Reply:
TODO: add a safe guard to make sure vec env only include the same type of scene